looper/submission_manager.py

```diff
@@ -13,6 +13,7 @@
     create_looper_args_text, grab_project_data, sample_folder
 
 from peppy import Sample, VALID_READ_TYPES
+from divvy.utils import write_submit_script
```

Comment:
Within `divvy`, can we have the package `__init__` file import this function, so that `divvy`-dependent packages (like `looper`) can import directly from `divvy` rather than needing to specify the subpackage or module? It makes it easier to reorganize the upstream package without needing to modify imports in dependent packages. Not a big deal since `looper` is also ours, but it could surprise hypothetical future package developers that decide to depend on `divvy` if a new release came out and broke some  of their imports

Reply:
Sure, makes sense